backend/src/main/java/ca/gov/dtsstn/cdcp/api/service/UserService.java

```diff
@@ -90,16 +98,20 @@ public ConfirmationCode createConfirmationCodeForUser(String userId) {
 	}
 
 	public Subscription createSubscriptionForUser(String userId, Subscription subscription) {
+		log.debug("subscription {}", subscription.getLanguage());
```

Comment:
You should make this log message a little more descriptive.. as is, this will just log the subscription data with no context around what the application was doing at the time.

Maybe something like "Creating new [{code}] subscription for user [{id}]"

Reply:
or `Creating new [{code}] subscription with language [{lang}] for user [{id}]`

Reply:
removed, it is for my own debugging